DigitalLearningSolutions.Web/ViewModels/LearningMenu/TutorialViewModel.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningMenu
+{
+    using DigitalLearningSolutions.Data.Models.TutorialContent;
+
+    public class TutorialViewModel
+    {
+        public TutorialContent TutorialContent { get; }
+        public int CustomisationId { get; }
+        public int SectionId { get; }
+
+        public TutorialViewModel(TutorialContent tutorialContent, int customisationId, int sectionId)
+        {
+            TutorialContent = tutorialContent;
+            CustomisationId = customisationId;
+            SectionId = sectionId;
+        }
+
+        public string ParseMinutes(int minutes)

```

Comment:
Does this need to be public? 🙂

Reply:
No, thanks for that! Changed in 0262ae6 🙂